{{cookiecutter.project_slug}}/pyproject.toml

```diff
@@ -72,6 +72,11 @@ mkdocs-literate-nav = ">=0.4.1,<1"
 mkdocs-macros-plugin = ">=0.6.3,<1"
 mkdocs-material = ">=8.2.1,<9"
 mkdocstrings = ">=0.17.0,<0.18.0"
+Sphinx = "^4.4.0"
```

Comment:
I think you should add conditionals here if the user wants mkdocs, sphinx or jupyter-book
but you can do it as a follow -up PR